resources/public/api/conf/1.0/schemas/sautr-details-response.json

```diff
@@ -1,32 +1,132 @@
 {
   "type": "object",
+  "required": [
+    "taxPayerType",
+    "taxPayerName",
+    "address"
+  ],
   "properties": {
-    "details": {
-      "type": "array",
-      "items": {
-        "type": "object",
-        "properties": {
-          "year": {
-            "type": "number",
-            "example": "2019"
-          },
-          "turnover": {
-            "type": "number",
-            "example": "1000000.00"
-          },
-          "registrationDate": {
-            "type": "string",
-            "example": "2001-01-01"
-          },
-          "taxSolvencyStatus": {
-            "type": "string",
-            "description": "S = Solvent, I = Insolvent, A = Administration Order, V = Voluntary Arrangement",
-            "enum": [
-              "S",
-              "I",
-              "A",
-              "V"
-            ]
+    "taxPayerType": {
+      "type": "string",
+      "description": "Individual, Partnership",
+      "enum": [
+        "Individual",
+        "Partnership"
+      ]
+    },
+    "taxPayerName": {
+      "type": "string",
+      "description": "The name of the organisation.",
+      "example": "Example Partnership"
+    },
+    "address": {
+      "type": "object",
+      "description": "The organisation's address.",
+      "required": [
+        "addressLine1",
+        "addressLine2",
+        "postcode"
+      ],
+      "properties": {
+        "addressLine1": {
+          "type": "string",
+          "description": "The first line of the organisation's address.",
+          "example": "123 Long Road"
+        },
+        "addressLine2": {
+          "type": "string",
+          "description": "The second line of the organisation's address.",
+          "example": "Some City"
+        },
+        "addressLine3": {
```

Comment:
Missing description, check individuals as address descriptions have been solidified in there. Check addresse on all schemas in here please.